docs/getting_started/requirements.md

```diff
@@ -46,7 +46,7 @@ For security and performance we generally recommend (unless otherwise noted) usi
 -   DBMS
     -   MySQL 5.7 or 8.0
     -   MariaDB 10.2, 10.3, 10.4
-    -   PostgreSQL 10+
+    -   PostgreSQL 12.2
```

Comment:
Hmm, we only support 12.2? On "other supported setups" we tend to be more flexible as long as something is within technical minimum requirements.